packages/core/src/Dialog/Dialog.styled.tsx

```diff
@@ -213,6 +214,7 @@ export const DialogContent = ({
           overflowX={overflowX}
           overflowY={overflowY}
           sheet={sheet}
+          hasFooterContent={footerContent}
```

Comment:
Just need to add a prop type for `hasFooterContent`

Reply:
I just checked and this is an internal only prop and TS is not throwing warnings/errors on build/lint